### PR TITLE
Foot Solutions #3053

### PR DESCRIPTION
**Pull request by adzika**
Added foot solutions shop and created Wikidata entry, there's no Wikipedia for this shop. I'm not sure how to add this to "healthcare" as I haven't found this label.

### CODE REVIEW

**Comment by adzika** [comment]
Thank you for your reply, it's one of my first pull requests, hence the mistakes, even though I tried to read docs thoroughly. :)

Ad 1. Yes, it's a WebStorm file and sure, I will add it to .gitignore.
Ad 2. Ok, I just wasn't sure how to do it because I didn't see that info in the docs. Maybe after I'll change the code, I can also add one sentence about leaving the Wikipedia entry empty in the docs if the website is not there?
Ad 3. Sure! Again I was trying to follow the docs but the animation looked quite different from what I saw on the screen and it was my first time doing this, so I got a bit confused.

I'm going to change all that now.

**Comment by adzika** [comment]
I've pushed the second commit, not sure if I need to anything else now?

**Comment by bhousel** [comment]
> I've pushed the second commit, not sure if I need to anything else now?

I still see the files under `.idea` on the pull request, but the other items look ok..

I think you can probably remove them with something like `git rm -R .idea`

**Comment by adzika** [comment]
Thanks, I deleted .idea files.

**Comment by bhousel** [comment]
Thank you, looks great @adzika 👍

**Comment by adzika** [comment]
Thanks for guiding me along the way. :)

File: brands/shop/shoes.json [hunk 1]
```diff
@@ -294,6 +294,14 @@
       "shop": "shoes"
     }
   },
+  "shop/shoes|Foot Solutions": {
+    "tags": {
+      "brand": "Foot Solutions",
+      "brand:wikidata": "Q69982369",
+      "name": "Foot Solutions",
+      "shop": "shoes"
```
Comment (1ec5) [inline brands/shop/shoes.json]: > I'm not sure how to add this to "healthcare" as I haven't found this label.

You can add any tags you need to as properties in this JSON object, alongside `shop`, `name`, etc. I was suggesting adding a `healthcare=something` or `healthcare:speciality=something` yah but wasn’t sure exactly what to set it to. Later on, as I fleshed out the Foot Solutions item on Wikidata to save it from deletion, I learned of the word “pedorthist”, which would work as a value for the `healthcare:speciality` tag.

Comment (1ec5) [reply brands/shop/shoes.json]: Added in 6a25dbc04ff82a35fd49f0d73b247dc099851d82 along with `shoes=orthopaedic`, which is [documented on the wiki](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dshoes).

**Pull request merged**